sdk/src/main/scala/com/horizen/account/state/AccountStateView.scala

```diff
@@ -137,8 +179,8 @@ class AccountStateView(metadataStorageView: AccountStateMetadataStorageView, mes
   // account specific getters
   override def getAccount(address: Array[Byte]): Account = ???
 
-  override def getBalance(address: Array[Byte]): Long = ???
+  override def getBalance(address: Array[Byte]): java.math.BigInteger = stateDb.getBalance(address)
```

Comment:
Maybe also getBalance should return a Try, as the other methods do

Reply:
Done